net.crossfireatlas.crossfire.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: net.crossfireatlas.crossfire
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: org.freedesktop.Sdk.Extension.openjdk11
+command: net.crossfireatlas.crossfire.start.sh
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: 
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: Crossfire_client
+    buildsystem: simple
+    build-commands: 
+      - install -D download /app/bin/jxclient.jar
+      - install -D net.crossfireatlas.crossfire.start.sh /app/bin/net.crossfireatlas.crossfire.start.sh
+      - install -D net.crossfireatlas.crossfire.desktop /app/share/applications/net.crossfireatlas.crossfire.desktop
+      - install -D net.crossfireatlas.crossfire.appdata.xml /app/share/metainfo/net.crossfireatlas.crossfire.appdata.xml
```

Comment:
you still don't install an icon.

And as I said, `.ico` isn't a valid icon format. PNG (128x128 at least)  or SVG are only accepted.

Reply:
OK, I have now the PNG (128x128) included (and references in the yaml file).